rpcserver.go

```diff
@@ -809,15 +814,151 @@ func (r *rpcServer) VerifyProof(ctx context.Context,
 	)
 	valid := err == nil
 
-	// TODO(roasbeef): also show additional final resting anchor
-	// information, etc?
+	rpcProof, err := marshalProofFile(proofFile, -1)
+	if err != nil {
+		return nil, fmt.Errorf("unable to marshal proof: %w", err)
+	}
 
-	// TODO(roasbeef): show the final resting place of the asset?
-	return &tarorpc.ProofVerifyResponse{
-		Valid: valid,
+	return &tarorpc.VerifyProofResponse{
+		Valid:        valid,
+		DecodedProof: rpcProof,
 	}, nil
 }
 
+// DecodeProof attempts to decode a given proof file that claims to be anchored
+// at the specified genesis point.
+func (r *rpcServer) DecodeProof(ctx context.Context,
+	in *tarorpc.DecodeProofRequest) (*tarorpc.DecodeProofResponse, error) {
+
+	if len(in.RawProof) == 0 {
+		return nil, fmt.Errorf("proof file must be specified")
```

Comment:
nit: we can directly specify the rpc field.

Reply:
Prefer leaving unchanged to stay consistent with VerifyProof